db/sqldb/cursor.go

```diff
@@ -7,8 +7,8 @@ import (
 	"github.com/RMI/pacta/db"
 )
 
-func offsetToCursor(i int) string {
-	return fmt.Sprintf("%d", i)
+func offsetToCursor(i int) db.Cursor {
+	return db.Cursor(fmt.Sprintf("%d", i))
```

Comment:
Sorry, didn't notice this before:

```suggestion
	return db.Cursor(strconv.Itoa(i))
```

It's also 5x faster...not that performance matters here, but think of all the μL CO2 emissions we're preventing

Reply:
Good catch.